qa/migration-tests/src/test/java/io/camunda/it/migration/util/MigrationITExtension.java

```diff
@@ -204,8 +226,8 @@ private void awaitImportersFlushed() {
                     && migrationDatabaseChecks.checkImportPositionsFlushed(TASKLIST));
   }
 
-  private void awaitProcessMigrationFinished() {
-    final var logger = (Logger) LoggerFactory.getLogger(ProcessMigrator.class);
+  private void awaitMigrationsFinished() {
```

Comment:
❓ Could this use the `AsyncMigrationsFinishedEvent` to check that they finished?
I feel like the `assertThat(appender.logs.size()).isGreaterThan(0)` is not the most robust way.

I know that this wasn't introduced by this PR though.